content/applications/general/companies/digest_emails.rst

```diff
@@ -2,82 +2,152 @@
+email and click on the :guilabel:`KPIs` tab. To add a :abbr:`KPI (key performance indicator)` to
+the digest email simply tick the checkbox next to the desired :abbr:`KPI (key performance
+indicator)`. After all :abbr:`KPIs (key performance indicators)` are added or deselected, then
+:guilabel:`Save` the progress.
+
+Recipients
+==========
+
+Recipients are added from the :guilabel:`Recipients` tab. To add a recipient simply navigate to
+:menuselection:`Settings --> General Settings --> Statistics` and click :guilabel:`Configure Digest
+Emails`. Then open the digest email and click on the :guilabel:`Recipients` tab. To add a recipient
+click :guilabel:`Add a line` and a pop-up will appear with available users to add on as recipients.
+Tick the checkbox next the the :guilabel:`Name` of the user and click the :guilabel:`Select` button.
+
+To remove a user as a recipient click the :guilabel:`❌` icon to the far right of the user listed in
+the :guilabel:`Recipients` tab.
+
+:guilabel:`Save` the work to implement the changes.
+
 .. _digest-emails/custom-emails:
 
-Custom digest emails
-====================
+Create a new digest email
+=========================
 
-To do so, click on **Configure Digest Emails** and then **Create**.
+To create a new digest email navigate to :menuselection:`Settings --> General Settings -->
+Statistics` and click :guilabel:`Configure Digest Emails`. Click :guilabel:`Create` to create a new
+digest email.
 
-From there, give your Digest Email a title, specify periodicity, and choose your desired KPIs and
-recipients fields as needed.
+A popup window appears and presents a variety of editable settings, which include:
 
-After you click **Save**, your new custom Digest Email is available as a selection in the **General
-Settings** dropdown menu.
+- :guilabel:`Digest Name` - the name of the Digest Email.
+- :guilabel:`Periodicity` - control the regimen in how often Digest Emails are sent.
+- :guilabel:`Next Send Date` - the date which the digest email will be sent again.
+- :guilabel:`KPIs` tab - check/uncheck each calculated KPI that appears in Digest Emails.
+- :guilabel:`Recipients` tab - add/remove users who receive the Digest Emails.
+
+From there, give the digest email a :guilabel:`Digest Name`, specify :guilabel:`Periodicity`, and
+choose the desired :guilabel:`KPIs` and add :guilabel:`Recipients` as needed.
+
+After clicking :guilabel:`Save`, the new custom digest email is available as a selection in the
+:guilabel:`General Settings` drop-down menu.
 
 .. _digest-emails/custom-kpi:
 
 Custom KPIs with Studio
------------------------
+=======================
+
+The :abbr:`KPIs (key performance indicators)` can be customized using Odoo *Studio*. Additional
+costs to the database subscription will be incurred should *Studio* need to be installed.
+
+For either *Your Odoo Periodic Digest* or a custom digest email, KPIs can be added by using Odoo
```

Comment:
```suggestion
For either *Your Odoo Periodic Digest* or a custom digest email, KPIs can be added by using Odoo
```
I've noticed that `KPIs` throughout this doc sometimes uses the abbr tag, sometimes not - I think it would be good to make sure it is consistent throughout